mongomock/aggregate.py

```diff
@@ -1364,6 +1365,31 @@ def _handle_sample_stage(in_collection, unused_database, options):
     return shuffled[:size]
 
 
+def _handle_sort_by_count_stage(in_collection, unused_database, options: Union[str, Dict]):
+    key_count = {}
+    sort_by_count = []
+
+    if isinstance(options, Dict) and '$mergeObjects' in options:
+        raise NotImplementedError(
+            "Although '$mergeObjects' is a valid operator for the $sortByCount stage, "
+            'it is currently not implemented in Mongomock.'
+        )
+    field_to_count = options[1:]
```

Comment:
It does the same thing, but it might be explaining a little better what is going on
```python
field_to_count = options.lstrip("$")
```